### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by MichalTuleja**
### Review checklist

* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: docs/prerequisites-linux.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Git (<https://github.com/git/git>) -- Used for cloning and updating Lisk Explorer
+
+  ```bash
+  sudo apt-get install -y git
+  ```
+
+- Tool chain components -- Used for compiling dependencies
+
+  ```bash
+  sudo apt-get install -y python build-essential automake autoconf libtool
```
Comment (Tschakki) [inline docs/prerequisites-linux.md]: I needed to install an additional package in order to run `nom install` successfully on Ubuntu: `libpng-dev`

File: docs/run_from_source.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+cd lisk-explorer
+```
+
+## Build Steps
+
+The NPM and Bower tools install all JavaScript dependencies needed to run the Lisk Explorer.
+
+```
+npm install
+bower install
```
Comment (Tschakki) [inline docs/run_from_source.md]: Bower is not needed anymore

File: docs/prerequisites-linux.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	  wget https://github.com/fiorix/freegeoip/releases/download/v3.4.1/freegeoip-3.4.1-linux-amd64.tar.gz
+	  tar -zxf freegeoip-3.4.1-linux-amd64.tar.gz
+	  ln -s freegeoip-3.4.1-linux-amd64 freegeoip
+	  nohup ./freegeoip/freegeoip > ./freegeoip/freegeoip.log 2>&1 &
+	```
+
+- PM2 (https://github.com/Unitech/pm2) -- PM2 manages the node process for Lisk Explorer and handles log rotation (Highly Recommended)
+
+  ```bash
+  sudo npm install -g pm2
```
Comment (Tschakki) [inline docs/prerequisites-linux.md]: No sudo needed here

File: README.md [hunk 4]
```diff
@@ -1,171 +1,48 @@
...
-To stop Explorer after it has been started with `PM2`, issue the following command:
-
-`pm2 stop lisk-explorer`
-
-## Docker
-
-### Using docker-compose
+You will also need Docker Compose [Install Docker Compose](https://docs.docker.com/compose/install/).
 
 Update `docker-lisk-core.env` to choose your preferred node. You can easily switch between Mainnet and Testnet nodes by changing content of the env file.
```
Comment (Tschakki) [inline README.md]: needs to be updated analog to docker setup page

File: docs/prerequisites-linux.md (hunk 1 above)
Comment (MichalTuleja) [reply docs/prerequisites-linux.md]: Done, thanks for pointing it out

File: docs/run_from_source.md (hunk 2 above)
Comment (MichalTuleja) [reply docs/run_from_source.md]: :+1:

File: docs/prerequisites-linux.md (hunk 3 above)
Comment (MichalTuleja) [reply docs/prerequisites-linux.md]: Updated.

File: README.md (hunk 4 above)
Comment (MichalTuleja) [reply README.md]: The new readme does not mention this method

**Pull request merged**